CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where author list parser can't generate bibtex for Chinese author. [#4169](https://github.com/JabRef/jabref/issues/4169)
 - We fixed an issue where the list of XMP Exclusion fields in the preferences was not be saved [#4072](https://github.com/JabRef/jabref/issues/4072)
 - We fixed an issue where the ArXiv Fetcher did not support HTTP URLs [#4367](https://github.com/JabRef/jabref/pull/4367)
+- We fixed an issue where the rank submenus would only show the first star icon [#4384](https://github.com/JabRef/jabref/pull/4384)
```

Comment:
Please remove the changelog entry. The issue was introduced in the current development version and thus fixing it is not a real change visible to the user (that stick to the released versions).

Reply:
Removed, makes sense. Thanks for the quick review.